src/file.ts

```diff
@@ -35,13 +35,15 @@ async function zipBuildResult(buildResult: BuildResult): Promise<void> {
 }
 
 async function moveBuildsToExportDirectory(buildResults: BuildResult[], moveArchived?: boolean): Promise<void> {
-  const fullExportPath = path.resolve(RELATIVE_EXPORT_PATH);
-  core.startGroup(`Moving exports to ${fullExportPath}`);
-
-  await io.mkdirP(fullExportPath);
-
   const promises: Promise<void>[] = [];
   for (const buildResult of buildResults) {
+    const fullExportPath = path.resolve(
+      USE_PRESET_EXPORT_PATH ? path.dirname(buildResult.preset.export_path) : RELATIVE_EXPORT_PATH,
```

Comment:
Definitely will want to document that the new property takes precedence over `relative_export_path`

Reply:
Yes, I realized this too and fixed it in the most recent commit 3f0aed2025956d904bb1ef439d214f59de970716